### PR TITLE
added cookie property in api body

### PR DESCRIPTION
**Pull request by TheNoumanDev**
Ticket: https://github.com/EnsembleUI/ensemble/issues/1649

Example EDL:
```yaml
getProducts:
    url: https://postman-echo.com/post
    method: POST
    manageCookies: true  # default is false
    onResponse: |
      console.log(response.body);
      console.log(response.headers['set-cookie']); // just to confirm
      console.log(response.cookies); 
```

### CODE REVIEW

**Comment by kmahmood74** [comment]
@TheNoumanDev I thought this was done, why is this PR still open?

**Review by kmahmood74 — APPROVED** [review]

**Comment by TheNoumanDev** [comment]
> @TheNoumanDev I thought this was done, why is this PR still open?

Was waiting for a review from Vu as you said Vu has better understanding of API understanding, But yeah will update sink,docs and schema and merge it.

**Pull request merged**
